app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers.rb

```diff
@@ -118,6 +118,10 @@ def snapshots
     @snapshots ||= snapshots_api.pcloud_cloudinstances_snapshots_getall(cloud_instance_id).snapshots || []
   end
 
+  def provider_region
+    @provider_region = cloud_manager.pcloud_location(connection)
```

Comment:
You probably want to ||= this

Reply:
Oops, you're right. Just force pushed a new commit with it fixed.